### DISCUSSION TITLE
PnP Search Results: display Today - Created date

### DISCUSSION THREAD

**Discussion by ileana2019** [microsoft-search/pnp-modern-search#1785] [Q&A]
Hello,
Is it possible to display a column inside the Search Results webpart as being the date difference: Today - Created date? I see you can use the [moment handlebar](https://github.com/helpers/handlebars-helpers#moment) but I just cannot make it work. Example: {{moment Created subtract=timeago format="DD MMMM YYYY"}};

Thank you!

**Comment by wobba** [comment]
You can try to play with something like the below, inspired from my response at https://github.com/microsoft-search/pnp-modern-search/discussions/1611#discussioncomment-1775214

```
Days: {{divide (subtract (moment 'X') (getDate item.Created 'X')) 86400}}
``` 

We would have to add more parameter support to have duration supported.
